cmslitemetadata_to_redshift/cmslite_groups_gdx.json

```diff
@@ -6,6 +6,7 @@
   "doc": "cmslite.csv.*",
   "dbschema": "cmslite",
   "dbtable": "metadata",
+  "ddl": "cmslite.to_redshift.sql",
```

Comment:
To keep consistency for variable names in all microservices, can we use same name"sql_query" here and probably use it in a same way. Open to discussion on this

URL - https://github.com/bcgov/GDX-Analytics-microservice/blob/main/s3_to_redshift/README.md

Reply:
added commit to match these standards